backend/src/core/utils/attachment.ts

```diff
@@ -0,0 +1,12 @@
+import { UploadedFile } from 'express-fileupload'
+
+const ensureAttachmentsFieldIsArray = (
+  attachments: UploadedFile | UploadedFile[]
```

Comment:
Why do we have to declared `UploadedFile` type ah? We're returning array for all cases

Reply:
Oh this is the argument type, not the return type. Yup returning array for all cases. 

Strictly the UploadedFile type is not necessary la, but I thought it be good to show which library this is meant to be working with